streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -1075,92 +1074,6 @@ public void shouldNotMakeStoreAvailableUntilAllStoresAvailable() throws Exceptio
 
     }
 
-    @Test
-    @Deprecated //A single thread should no longer die
-    public void shouldAllowToQueryAfterThreadDied() throws Exception {
```

Comment:
Why are we removing this test? We should just update the test to use the new handler instead.